frontend/src/pages/AccountReceive.vue

```diff
@@ -101,6 +101,14 @@
         </div>
         <div v-else class="text-center text-italic">{{ $t('Receive.fetching-latest') }}</div>
       </div>
+      <div>
+        <base-button
```

Comment:
What is the best way to test this? I can't get it to show on my receive page.

Reply:
Do you mean you don't see the `Stop` button at all?

Reply:
I don't see it on the table view, don't know what scenario should trigger it.

![image](https://github.com/ScopeLift/umbra-protocol/assets/12705146/b33f8d48-0ca9-4be3-9206-32e9bf5e6ce0)

Reply:
The video in the description of this PR should give you some idea, but basically the button should show up when the announcements are being fetched or being scanned.